packages/admin-ui/src/common/types.ts

```diff
@@ -371,6 +371,7 @@ export interface PortMapping extends BasicPortMapping {
   ephemeral?: boolean;
   ip?: string;
   deletable?: boolean;
+  hasIpv6?: boolean;
```

Comment:
If there is no-one consuming this property remove. We should add features only when useful

Reply:
I know, but who knows in the future?

Reply:
removed type

Reply:
> I know, but who knows in the future?

Then we'll add it in the future when necessary